src/include/btmem.h

```diff
@@ -758,6 +773,22 @@ struct __wt_page {
 
     uint64_t cache_create_gen; /* Page create timestamp */
     uint64_t evict_pass_gen;   /* Eviction pass generation */
+
+#ifdef HAVE_DIAGNOSTIC
+    WT_SPLIT_PAGE_HIST split_hist[4];
```

Comment:
Instead of the magic number 4, can we do the same as the ref hist?
```
#define WT_REF_SAVE_STATE_MAX 3
```
I don't have a strong preference between 3 and 4. Is there a specific reason why you chose 4?

Reply:
I think 3 is fine, there's no particular reason I had 4 here.

Reply:
I previously had the number #defined but s_all complains about a single use of a definition.

Reply:
Nevermind, I'm not sure what problem I was having with s_all. Fixed now.